include/ibamr/INSHierarchyIntegrator.h

```diff
@@ -539,9 +539,11 @@ class INSHierarchyIntegrator : public IBTK::HierarchyIntegrator
      */
     SAMRAI::tbox::Pointer<IBTK::CartGridFunction> d_U_init, d_P_init;
     SAMRAI::solv::LocationIndexRobinBcCoefs<NDIM> d_default_bc_coefs;
-    std::vector<SAMRAI::solv::RobinBcCoefStrategy<NDIM>*> d_bc_coefs, d_U_bc_coefs, d_U_star_bc_coefs;
+    std::vector<SAMRAI::solv::RobinBcCoefStrategy<NDIM>*> d_bc_coefs, d_U_bc_coefs;
+    std::vector<SAMRAI::solv::RobinBcCoefStrategy<NDIM>*> d_U_star_bc_coefs;
```

Comment:
Is this supposed to be a `unique_ptr`?

Reply:
No - that requires some more work since inheriting classes reset that pointer. I am not trying to remove all `new`/`delete` pairs here, just some.